keyboards/candybar/rules.mk

```diff
@@ -36,14 +36,14 @@ DFU_SUFFIX_ARGS = -p DF11 -v 0483
 # Build Options
 #   comment out to disable the options.
 #
-EXTRAFLAGS+=-flto
+# EXTRAFLAGS+=-flto
```

Comment:
I'm curious, why disable LTO?

Reply:
It wasn't needed after disabling console and command. I can try leaving it enabled.

Reply:
Okay, that's fine.  It's just that it does reduce the size a good amount.

Also, it may be testing with `LINK_TIME_OPTIMIZATION_ENABL = yes`, as that enables LTO, and disables the old macro and functions features.